### ISSUE TITLE
action-bar action-title

### ISSUE THREAD

**Issue by erjdriver** [NativeScript/theme#112]
When the active theme is dark, action-title text is colored black and almost impossible to see.

`<Label text="{{ getAppName() }}" class="action-title"></Label>`

The color for `.form .input-field .label` is too light under the light theme - difficult to see.

What's the recommended way to make mods but still be able to update to the latest version of the theme.

**Comment by NathanWalker** [comment]
Fix published with 0.2.5 👍

Thanks for reporting @erjdriver . `action-title` now adjusts when using light or dark skin.
I also darkened `.form .input-field .label` just a tad. 

The recommended way to customize the theme is to use `SASS` by installing:
```
npm i nativescript-dev-sass --save
```
Then rename your `.css` to `.scss` and setup your sass like so:
```
// dark variables
@import 'nativescript-theme-core/scss/dark';

// Customize any of the theme’s variables here, for instance $btn-color: red;

// Import the theme’s main ruleset.
@import 'nativescript-theme-core/scss/index';

// Place any CSS rules you want to apply on both iOS and Android here.
// This is where the vast majority of your CSS code goes.
```

For a full list of SASS variables that can be customized, refer here:
https://github.com/NativeScript/theme/blob/master/app/scss/_variables.scss

**Comment by erjdriver** [comment]
thanks - i'll try this out.

While I'm testing out the themes - I've got a settings page and naturally I set the main page css class to "form" - correct?

But this class doesn't set the background color to the theme background color - which I think should be the dominant theme specific color.

If the theme background color is set - then the other control colors can be left light and it should work out.

Thanks.
